partner-chains-cli/src/cardano_key.rs

```diff
@@ -0,0 +1,48 @@
+use crate::IOContext;
```

Comment:
See register2.rs and prepare_chain_params.rs changes below.
Extracted in this PR, because register1.rs parses cardano key file as well.